src/ex2/RenderList.js

```diff
@@ -0,0 +1,49 @@
+export default class RenderTaskList {
+  constructor(tracker, tasks) {
+    this.todoList = document.querySelector(".todo-list");
+    this.pendingTasks = document.querySelector(".upcomingTasks");
+    this.tasksList = tasks;
+    this.tracker = tracker;
+  }
+  render(list) {
+    this.todoList.innerHTML = "";
+    list.forEach((v, index) => {
+      const listItem = this.renderItem(v, index);
```

Comment:
wouldn't call this function `renderItem` as it's not rendering the item, but creating the dom element